### DISCUSSION TITLE
Different HUD for each mods (Classic, taiko, Mania & catch)

### DISCUSSION THREAD

**Discussion by KiwiKin1** [ppy/osu#19685] [Ideas] [answered]
Add support for **customizing each game mods HUD in skin layout editor**, which make it **easy for looking speed meter/accuracy... for each game mode** & look better without having same skin component placement for all game mode.

_As the image attached, there were some difficulty for me cant have individual hud for each game mode, making some skin component hard to see..._
![ideaimage2](https://user-images.githubusercontent.com/106151046/183882711-112f1b6c-a8d7-47bb-a7dd-2588baf2d950.png)
![ideaimage1](https://user-images.githubusercontent.com/106151046/183882719-4688164e-e5c8-4ce8-b6c2-143e636a7040.png)
![ideaimage3](https://user-images.githubusercontent.com/106151046/183882725-d3a90804-6710-4a0a-be7c-35805478434c.png)

started by KiwiKin from osu!lazer

**Comment by peppy** [comment, marked as answer]
We do have plans for more per-ruleset skin specifications in the future.

**Reply by KiwiKin1** [reply]
anyways i have an issue about taiko skinning too, is it ok i discuss here or start new issue report on the issue tab?

**Reply by peppy** [reply]
Please open a new discussion.

**Reply by KiwiKin1** [reply]
ok thx for answering and helping me :)
